.github/workflows/test.yml

```diff
@@ -114,7 +114,7 @@ jobs:
           fi
 
   yarn-test:
-    runs-on: ubuntu-latest
+    runs-on: larger-runner
```

Comment:
oh dang, how come?

would it make sense to put the e2e on a separate job?

Reply:
+1, these cost more than $1 each run. We want to run only the bare minimum on them

Reply:
Good to know! i'll try this without the larger-runner